app/models/lookup_key.rb

```diff
@@ -7,6 +7,7 @@ class LookupKey < ActiveRecord::Base
 
   KEY_DELM = ","
   EQ_DELM  = "="
+  VALUE_REGEX = Regexp.new("\\A[\\w\\s]+#{EQ_DELM}[\\w\\s]+(#{KEY_DELM}[\\w\\s]+#{EQ_DELM}[\\w\\s]+)*\\Z")
```

Comment:
I think the `[\w\s]` is too restrictive here, and it's (quietly!) preventing me from saving a matcher of: `hostgroup=Foo/Bar`, since `/` isn't in the word character class.  I'd suggest being a lot more liberal, accepting anything except KEY_DELM perhaps?